Resources/Catalog/FileCatalog.py

```diff
@@ -15,11 +15,13 @@ class FileCatalog( object ):
   ro_methods = ['exists', 'isLink', 'readLink', 'isFile', 'getFileMetadata', 'getReplicas',
                 'getReplicaStatus', 'getFileSize', 'isDirectory', 'getDirectoryReplicas',
                 'listDirectory', 'getDirectoryMetadata', 'getDirectorySize', 'getDirectoryContents',
-                'resolveDataset', 'getPathPermissions', 'getLFNForPFN', 'getUsers', 'getGroups', 'getFileUserMetadata']
+                'resolveDataset', 'getPathPermissions', 'getLFNForPFN', 'getUsers', 'getGroups', 'getFileUserMetadata',
+                'getMetadataFields', 'findFilesByMetadata']
```

Comment:
You are right. But this will not work if the methods are translated to the server directly by the RPCClient, it will always return True. LcgFileCatalogClient should be OK, but for DIRAC services is not. Any idea ?